tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -59,6 +59,7 @@ static PyObject * THPVariable__is_view(PyObject *self, PyObject* args)
   END_HANDLE_TH_ERRORS
 }
 
+// binded here bc no support for first-class functions in native_functions.yaml
```

Comment:
nit: The word "binded" might not mean much to someone who does not know much about the codebase. It would be great to provide a little more context.

"binded here (instead of being declared in native_functions.yaml) because native_functions doesn't support first-class functions.

ditto for the other comments as well

Reply:
Yea, that's fair. 
Maybe: "implemented on the python object bc no support for first-class functions in native_functions.yaml. See: ATen/native/README.md for more context" ?

Reply:
go for it